### ISSUE TITLE
Taking over maintenance

### ISSUE THREAD

**Issue by pixeltrix** [luislavena/mysql-gem#33]
Hi @luislavena 

As suggested in the README, I'm opening an issue for taking over maintenance of this gem. My employers (@unboxed) maintain a number of older Rails apps for people and we're currently trying to get a Rails 3.2 app running on Ruby 2.3.1. As part of that I'm working on cleaning up the Rails test suite on 3-2-stable and the errno warning is bugging me :smile:

So what do I need to do next?

**Comment by luislavena** [comment]
Hello @pixeltrix, really sorry for the delayed response. Thought I sent a message before and realized never hit *Comment*. :cry: 

In relation to taking maintenance, I believe the first thing will be update the test suite and ensure test are passing against different versions of Ruby on both Travis and AppVeyor.

Once that is done, send a pull request, we get all green, merge and then I add you as contributor and give you push permissions on RubyGems.

Asking this way so we know you're familiar with the code and the workflow. Perhaps later we can transfer ownership to a different org to give you more freedom, but I guess we have to start somewhere :smile: 

Let me know if that works for you.

Cheers.

**Comment by pixeltrix** [comment]
@luislavena that all sounds fine - will take some time over the Christmas break to bring things up to speed.

**Comment by brauliobo** [comment]
isn't this completed yet?

**Comment by luislavena** [comment]
@brauliobo no, if you are offering to take over maintenance, this is your opportunity.

Cheers.

**Comment by brauliobo** [comment]
yes, I accept. At least for now I can redirect users to the `mysql2` and let them know that this gem is currently in need of fixes for ruby 2.4 and others issues, in the README file.

**Comment by luislavena** [comment]
@brauliobo you don't need to accept maintenance in order to send a PR with README improvements 😕 

If you want to update the documentation indicating this, simply open a PR (Pull Request) that **only** touched the README with the updated text, I can get that merged.

Anything else (code changes and such) will not be merged, as it requires more attention that I can provide to the project.

Hope you understand.

Cheers.

**Comment by brauliobo** [comment]
Sorry, I think some pull requests can be merged and tested after travis is setup. I can do it also. So making PRs won't make sense as you won't have time to review them.

**Comment by luislavena** [comment]
> So a PRs won't make sense as you won't have time to review them.

I think you're missing the point of the maintenance role, or I missed what you meant previously:

> At least for now I can redirect users to the mysql2 and let them know that this gem is currently in need of fixes for ruby 2.4 and others issues, in the README file.

If you want to only update `mysql` README, you can send a PR with README changes, that can be merged quickly without lot of review.

Anything else, code, testing, etc. will require time that I don't have available.

Please refer to my previous comment in relation to what is needed from anyone wanting to take ownership of the package:

https://github.com/luislavena/mysql-gem/issues/33#issuecomment-264207746

By doing those things you not only get familiarized with the codebase and project but also show the interest/investment you have into maintenance of the project.

So put it simple: not going to transfer maintenance of a project to anyone not willing to put that level of effort **upfront**. I cannot blindly give away permissions to push new versions of a gem that has huge dependency implications.

Hope you can understand.

Cheers.

**Comment by brauliobo** [comment]
ok, nop, I thought enabling travis, merging PRs that pass, and updating the README would be great for this project. hopefully someone can do better
